packages/react-lowcode/src/codegen/generation/generators/module-generator.ts

```diff
@@ -1,13 +1,22 @@
 import { PageComponent } from '../react-components/react-component-helper'
 import GenerationContext from '../context'
-import { TableGenerator, TableGeneratorFactory } from './table/table-generator-factory'
+import { TableGenerator, TableGeneratorFactory } from './index/table-generator-factory'
```

Comment:
let's not call a package/folder index (list could be better naming list-detail, master-detail...)